pkg/lorry/engines/models/replca_role_types.go

```diff
@@ -29,3 +31,10 @@ const (
 	LEARNER   = "Learner"
 	CANDIDATE = "Candidate"
 )
+
+// IsPrimaryForNormal returns true if the role is primary,
+// it is used for the case where db manager do not implemement the IsLeader method.
+// use it curefully, as it is for normal case, and may be wrong for some special cases.
```

Comment:
suggest rename it to "IsLikelyPrimaryRole", which implies the uncertainty of the function.